### PR TITLE
Update a bunch more images

### PR DESCRIPTION
**Pull request by potocnyj**
There are others that need fixing, but I'll ship these while I have time.

### CODE REVIEW

**Comment by potocnyj** [comment]
FYI @Preetam

File: content/how-to-use-vividcortex/metrics.md [hunk 1]
```diff
@@ -57,3 +57,5 @@ including a time-based representation of that metric and host information.
 ## Host Breakdown
 
 The metric's details page shows the hosts the metric has been associated with during the time selected.
```
Comment (Preetam) [inline content/how-to-use-vividcortex/metrics.md]: `metric's details`
🙄

Comment (potocnyj) [inline content/how-to-use-vividcortex/metrics.md]: Fixed, I updated it to match the language we use on the queries page

**Comment by Preetam** [comment]
@potocnyj Did you see https://github.com/VividCortex/docs/commit/d81ddeff866c13c7ced28bcc2b49c037136f982b#commitcomment-19136493?

**Comment by potocnyj** [comment]
I did, I can fix it in a followup PR but note that we do not capitalize `query details` on the Queries page either

**Comment by Preetam** [comment]
We wrote `Query Details` [here](https://www.vividcortex.com/product-updates/updated-query-details-page-and-more).

**Comment by potocnyj** [comment]
K, let's capitalize both then.

**Pull request merged**